package.json

```diff
@@ -39,9 +39,11 @@
   "devDependencies": {
     "babel-eslint": "^8.0.0",
     "dedent": "^0.7.0",
-    "eslint": "^4.12.1",
+    "eslint": "^4.14.0",
     "eslint-config-babel": "^7.0.1",
+    "eslint-old": "npm:eslint@4.13.1",
```

Comment:
Wow, I learned this notation today.

Reply:
Unfortunately, I think it's only supported by `yarn` and not `npm`.

Reply:
Oh, thank you..